data-tool/flows/corps_tombstone_flow.py

```diff
@@ -296,12 +297,22 @@ def update_auth(conn: Connection, config, corp_num: str, tombstone_data: dict):
         )
     if config.UPDATE_ENTITY:
         business_data = tombstone_data['businesses']
-        AuthService.create_entity(
+        entity_status = AuthService.create_entity(
             config=config,
             business_registration=business_data['identifier'],
             business_name=business_data['legal_name'],
             corp_type_code=business_data['legal_type']
         )
+        if entity_status == HTTPStatus.OK:
+            update_email_status = AuthService.update_contact_email(
+                config=config,
+                identifier=business_data['identifier'],
+                email=tombstone_data['admin_email']
+            )
+            if update_email_status != HTTPStatus.OK:
+                raise Exception(f"""Failed to update admin email in auth {business_data['identifier']}""")
+        else:
+            raise Exception(f"""Failed to create entity in auth {business_data['identifier']}""")
```

Comment:
@argush3 a bit hesitant here. Do you think it will be better to raise exception if auth updates fail? It will cause lear updates to rollback together.

Reply:
yes, we want to rollbacks in LEAR if auth api calls fail.  It will make it easier to reload businesses that way as opposed to having an business in LEAR but missing auth entries if we need it.  

Is this question about the contact email?  Or the `AuthService.create_entity` block above?

Reply:
This is question is for both create entity and update email. But what about the failture of creating affilication?

Reply:
i think all 3 need to rollback the LEAR data ideally.

Just a note that this is just useful for dev/test data loading purposes.  For the actual big data migration, we won't be calling auth api at all is what I'm working off of.  For the small number of early adopters we load, I think there is a good chance we create all the auth entries.